src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -179,9 +183,29 @@ def _fill_ftp_publishing_url(webapp, resource_group_name, name, slot=None):
     return webapp
 
 
+def _format_linux_fx_version(custom_image_name):
+    # set it as a space (Once fixed in backend, will actually set to empty here)
+    fx_version = ' '
+    # handles case of only spaces
+    if custom_image_name.strip():
+        if 'DOCKER|' not in custom_image_name:
```

Comment:
@LukaszStem, please also update the code using `.startswith`, `in` is not accurate